.github/workflows/build-publish.yaml

```diff
@@ -3,13 +3,13 @@ name: Build and Publish Docker image
 on:
   pull_request:
     branches:
-    - main
+      - main
```

Comment:
yamllint?

Reply:
Ahh no -- I think the Prettier extension I have installed in VSCode